src/keyValueStore.js

```diff
@@ -17,9 +17,22 @@ class KeyValueStore {
    * @param     {String}    key                     the key
```

Comment:
Could you add the optional param `opts.metadata` to the jsdoc here?

Reply:
Yep definitely, still wip. Just wanted to get some comments on whether I should maintain compatibility with the old format for non metadata queries or if they always should use the new response format.

Having the return be string|object could be confusing for people

Reply:
Yep, I definitely see the concern. However I think it's not a good idea to break backwards compatibility. 
We have the same quirk in the `Box.getProfile` function as well.